[QA][master] Unable to build client in drone-ci (LibreGraphAPI error)

### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the bug

```bash
-- Looking for lstat - found
CMake Error at src/libsync/CMakeLists.txt:1 (find_package):
  By not providing "FindLibreGraphAPI.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "LibreGraphAPI", but CMake did not find one.

  Could not find a package configuration file provided by "LibreGraphAPI"
  (requested version 1.0.4) with any of the following names:

    LibreGraphAPIConfig.cmake
    libregraphapi-config.cmake

  Add the installation prefix of "LibreGraphAPI" to CMAKE_PREFIX_PATH or set
  "LibreGraphAPI_DIR" to a directory containing one of the above files.  If
  "LibreGraphAPI" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
```

### Client version number

Master branch

--- a/.drone.star
+++ b/.drone.star
@@ -16,7 +16,7 @@ def main(ctx):
   #   'latest': '22.04'
   base_img_tag = {
     'latest': ['ubuntu', '22.04'],
-    'fedora': ['fedora', '36'],
+    'fedora': ['fedora', '38'],
   }
 
   config = {

--- a/fedora/Dockerfile.amd64
+++ b/fedora/Dockerfile.amd64
@@ -1,4 +1,4 @@
-FROM fedora:36
+FROM fedora:38
 
 LABEL maintainer="ownCloud DevOps <devops@owncloud.com>" \
   org.label-schema.name="ownCloud Client" \

--- a/fedora/Dockerfile.amd64
+++ b/fedora/Dockerfile.amd64
@@ -75,3 +75,13 @@ RUN git clone https://github.com/kdab/kdsingleapplication && \
     cd ../.. && \
     rm -r kdsingleapplication && \
     ldconfig
+
+RUN git clone https://github.com/owncloud/libre-graph-api-cpp-qt-client && \
+    cd libre-graph-api-cpp-qt-client/client && \
+    mkdir build && \
+    cd build && \
+    cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=/usr -G Ninja && \
+    ninja install -v && \
+    cd ../../.. && \
+    rm -r libre-graph-api-cpp-qt-client && \
+    ldconfig

--- a/latest/Dockerfile.amd64
+++ b/latest/Dockerfile.amd64
@@ -89,3 +89,13 @@ RUN git clone https://github.com/kdab/kdsingleapplication && \
     cd ../.. && \
     rm -r kdsingleapplication && \
     ldconfig
+
+RUN git clone https://github.com/owncloud/libre-graph-api-cpp-qt-client && \
+    cd libre-graph-api-cpp-qt-client/client && \
+    mkdir build && \
+    cd build && \
+    cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=/usr -G Ninja && \
+    ninja install -v && \
+    cd ../../.. && \
+    rm -r libre-graph-api-cpp-qt-client && \
+    ldconfig

